backend/scripts/test_llm_client.py

```
@@ -0,0 +1,141 @@
+    }
+
+    prompt = job_scoring_prompt(job_data=job_data, profile_data=profile_data)
+    score_payload = await client.generate_json(prompt=prompt)
+
+    required_fields = {"score", "category", "explanation"}
+    if not required_fields.issubset(score_payload):
+        raise AssertionError("Job scoring payload missing required fields")
+
+    score_value = score_payload.get("score")
+    if not isinstance(score_value, int) or score_value < 0 or score_value > 100:
+        raise AssertionError("Job scoring payload contains invalid score")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)